### DISCUSSION TITLE
Home Office é uma solução ou cilada?

### DISCUSSION THREAD

**Discussion by lucasmahle** [frontendbr/forum#733] [Discussão]
O que eu mais gosto de ver aqui no fórum, são perguntas sobre carreira, são os tópicos que mais fomentam contribuição. E por isso espero que contribuam comigo para resolver meu dilema pessoal.

Eu ingressei na universidade esse ano (em Agosto), e por ser federal, é bem intenso os estudos. O fato é que tenho 21 anos, moro sozinho (divido AP com amigo) meus pais são de outra cidade (60 km), trabalho o dia inteiro como programador e trabalho na área a quase 4 anos já. 
Eu me considero uma pessoa muito organizada, mas conforme o tempo passa, o curso fica mais complicado pelo volume de conteúdo passado, e por mim, abriria mão de parte do trabalho para me focar nos estudos. Mas como trabalho numa empresa que não tem muita flexibilidade quanto a horários, eu passei a ver home office não como um objetivo, mas como uma necessidade.

É viável eu procurar algo home office por esses aspectos? Ou é arriscado em termos financeiros? E quanto a facilidade pra conseguir home office?

**Comment by mathvbarone** [comment]
Acredito que o @LFeh seria a pessoa mais especializada em ajudar quando o assunto é home-office.

**Reply by ma12lopz** [reply]
Este blog é o seu balcão único para estratégias acionáveis ​​para ajudá-lo a ganhar dinheiro online e alcançar a liberdade financeira. Junte-se a nós para aprender como ganhar dinheiro online com renda passiva e alavancar investimentos. link: www.diaryofasidehustler.com

**Comment by willycamargo** [comment]
Sendo sincero: quantas das atividades que você exerce hoje realmente precisam da sua presença física no escritório?

Home office é um trabalho como outro qualquer. Empresas que trabalham com esse regime já sabem como lidar com os possíveis problemas e estão dispostas a ajudar o colaborador com a adaptação (se realmente quiser o profissional como colaborador, claro).

Obviamente algumas pessoas não possuem a disciplina necessária, mas não acredito que seja o seu caso, pois você mesmo disse que é organizado.

Meu feedback quanto ao trabalho remoto: foi difícil acostumar, hoje tenho meu ambiente montado com as ferramentas e conforto necessário para ser tão produtivo quanto sou no escritório e com a vantagem de não despender tanto tempo com transporte.

1. É viável financeiramente, não existem motivos para duvidar disso, apenas procure uma empresa séria e um cargo com seu perfil.

1. Não é fácil conseguir, os processos tendem a ser mais rigorosos e concorridos.

1. Na minha opinião o combo perfeito seria: empresa séria (já acostumada com esse tipo de regime) + disciplina e compromisso por parte do profissional.

**Comment by felipefialho** [comment]
Vou dar meus dois cents.

## Vale a pena?

Vale, vale sim. Mas depende.

A etapa da carreira/vida é muito importante nesse caso. Sou TOTALMENTE contra que profissionais não-seniors trabalhem remoto. Porque perdem uma etapa importante da carreira, onde desenvolveriam muito mais em contato com outros profissionais.

## Como foi comigo

Trabalhei remoto por 2 anos, entre 2015 e 2017. E sem dúvidas voltaria a trabalhar no futuro. Porém, no fim desse ciclo, eu já estava sentido que um contato mais direto com outros profissionais, estava começando a atrapalhar saltos maiores na carreira.

Porém, carreira não é tudo, e realmente a qualidade de vida é muito maior trabalhando home office. No meu caso, tentei diminuir isso indo morar perto do meu trabalho atual. Mas cada caso é um caso, depende muito das prioridades da pessoa naquele momento.

## Perfis

Outro ponto, é que por mais que vendam que home office é a bala de prata do trabalho moderno, eu discordo COMPLETAMENTE. 

Existem diferentes perfis, e nem todo se adequa ao trabalho remoto. As vezes falta foco, as vezes falta comprometimento, e a pessoa simplesmente não vai produzir. Isso vai causar danos para a empresa e para o próprio profissisonal.

___

Enfim, eu gosto, recomendo, mas com cuidado.

**Comment by victorferraz** [comment]
## Qualidade de vida
Acho que esse é o principal ponto, eu sou de Belo Horizonte e esse ano eu consegui ir para Argentina, Uruguay, São Paulo e agora estou em Curitiba, se eu não trabalhasse remoto eu nunca iria ter conseguido fazer essas viagens no meio de semana sem ter tirado férias.
Além do fato de você não perder o tempo no ônibus,  muitas vezes eu acordava fazia um café e já começava a trabalhar.


## Evolução profissional
Quando se trabalha fisicamente com uma equipe foda, é possível você evoluir muito mais do que trabalhar com essa mesma equipe remotamente, acho que isso é um ponto muito forte a ser pensando. Para quem trabalha remoto tem que focar o tempo livre em  estudos. Eu concordo com  o @LFeh   eu também não recomendaria trabalho remoto para as pessoas que estão começando, mas  existe alguns casos de pessoas que trabalharam quase a vida toda remoto e conseguiram se dar muito bem do mesmo jeito.


## Contato interpessoal
Essa parte é complicada, tinha época que eu ficava semanas sem conversar com alguém pessoalmente além da minha esposa, isso me incomodava bastante,  resolvi isso com alguns ajustes:
 - Começar a  organizar alguns meetups e eventos com mais frequência para fazer contato com outras pessoas. 
  - Ir trabalhar em coworkings  é algo muito legal para se fazer para conhecer outras pessoas de áreas diferentes. Ultimamente eu utilizo esse site [aqui](https://beerorcoffee.com)  para pagar um preço mais barato.


## Adaptação
Muita gente tem dificuldade de se adaptar, as vezes por se ter uma liberdade muito grande que acaba não conseguindo administrar muito bem o seu tempo, uma empresa muito organizada pode ajudar bastante nesse processo de adaptação. Conheço algumas pessoas que trabalham remoto, que acorda toma banho muda de roupa como se tivesse saindo para trabalhar, segundo elas isso ajuda no processo de adaptação  é como se tivesse dizendo para o seu cérebro que você está indo trabalhar.


## Como conseguir
Essa parte é um pouco complicada
 - para vagas internacionais é preciso focar no inglês (ler, falar e escutar)
 - estudar teste algorítimo.
 - se sentir confortável em participar de testes com telas compartilhadas, acho que é mais uma questão de confiança
 - estudar skills das vagas que você busca.
 - investir em network, uma indicação sempre ajuda na hora de conseguir um emprego, se no final da entrevista ficar entre você e um desconhecido, com certeza irão escolher você, pelo fator indicação. Ok mas como conseguir network? Você pode ir em meetups / eventos, contribuir com open source, que sem forçar a barra você começa a conhecer pessoas novas 
 - Sites;
[Github Jobs](https://jobs.github.com/positions?description=&location=Remote)
[Front End Br Vagas](https://github.com/frontendbr/vagas/issues?q=is%3Aissue+is%3Aopen+label%3A%22Remoto+-+Total%22)
[Wfh](https://www.wfh.io/)
[We Work Remotly](https://weworkremotely.com/)
[Staff](https://staff.com/)
[Stackoverflow jobs](https://stackoverflow.com/jobs/remote-developer-jobs)
[Remote Ok](https://remoteok.io/remote-jobs)
[Landing Jobs](https://landing.jobs/)
[Toptal](https://www.toptal.com/)

**Comment by lucasmahle** [comment]
O que vocês falaram foi muito bom, muito bom MESMO.
Não quero tornar a issue focado na minha situação, quero deixar ela mais generalizada. 
Mas mesmo assim preciso perguntar mais algo, principalmente referente ao que o @LFeh disse.
Eu tenho quase 4 anos de xp em dev. Eu me garanto quanto a organização de tempo (pessoal), mas em nenhum trabalho eu tive uma boa arquitetura de projetos, ou seja, um gerente que separe as tarefas, que defina o tempo, nada disso, só aquele modelo antigo do "cliente pediu", "aprovou orçamento", "faz ai até semana que vem".

No trabalho remoto, eu sei que há uma liberdade pessoal, mas e quanto à organização do projeto? Na maioria dos casos a equipe mantem contato ou é muito relativo?

Eu gosto da ideia de trabalhar remoto, mas não tenho noção por onde começar.

**Comment by lucasmahle** [comment]
Agora me surgiu mais uma dúvida. Sobre o modelo de trabalho.
Quando falamos de trabalho remoto, é mais comum achar trabalhos "fixos", aonde eu participo de uma equipe, que há reuniões etc, como se fosse no modelo padrão só que com flexibilidade e remuneração de acordo com desempenho?
Ou.
Buscar trabalhos por meio de freelas, sem tem cliente fixo, mas sim trabalhos específicos e sem uma necessária continuidade?

Qual é/era o modelo que vocês trabalham/trabalhavam?

**Reply by YasmimVieira** [reply]
Olá Lucas, quando trabalhei remoto era um trabalho fixo e ocorrida normalmente, reuniões, equipes e etc, mas era contratação PJ, por´me não acho dificil achar algo remoto sendo CLT e no caso do PJ, acho muito mais economico trabalhar de casa mesmo, por não ter beneficios ;)

**Comment by bernardodiasc** [comment]
Já fiz muito dos dois modelos, depende muito do profissional. A realidade é que a maioria dos profissionais não são preparados pra serem autônomos, seja por falta de experiência ou o próprio perfil profissional. Eu iniciei como autônomo e foi muito difícil por longos anos, só recomendo isso pra quem quer ir além mesmo, pra quem gosta de empreender.

Daí, como o @LFeh comentou, iniciar as experiências de trabalho remoto em times e contratos mais estáveis é a melhor opção pra quem ta começando (não na profissão, mas começando no trabalho remoto. Se estiver começando na profissão, vai pro escritório mesmo).

**Comment by bernardodiasc** [comment]
> No trabalho remoto, eu sei que há uma liberdade pessoal, mas e quanto à organização do projeto? 
> Na maioria dos casos a equipe mantem contato ou é muito relativo?

Não existe uma só resposta pra essas perguntas. Cada caso é um caso. Alguns clientes/gerentes são tão ruins que é melhor ficar longe, sério, não importa o dinheiro. Leva um tempo até ter um bom e lucrativo network, muitos erros e acertos.

**Comment by kazzkiq** [comment]
@lucasbento Acho que a melhor forma de manter controle da equipe sem transformar o home office em um Big Brother, é fazendo daily meetings estilo Scrum.

Fazendo um ponto diário de 10-15 minutos com a equipe permite que vocês tenham uma ideia do que foi feito no dia anterior, do que tá no escopo do dia atual, e quais as dificuldades que cada um tá tendo. Mesmo sendo meio informal, essa reunião diária acaba dando uma base pro que cada um vai fazer durante o dia.

**Comment by lucasbento** [comment]
cc @lucasmahle, obrigado pela resposta, @kazzkiq! ahaha

**Comment by kazzkiq** [comment]
Rapaz, acabei trocando os Lucas. Hangover de segunda... Sorry! 😄

**Comment by lucasmahle** [comment]
"CC" ASHUHASUHAS genial.
@kazzkiq e como eu consigo achar essas empresa? (N maneiras, eu sei, mas dê uma dica)
Você trabalha remoto ou já trabalho?
